components/tracker/src/lib.rs

```diff
@@ -85,13 +85,17 @@ impl Tracker {
     }
 }
 
-#[derive(Debug, Default)]
+#[derive(Debug, Default, Clone)]
 pub struct RequestInfo {
     pub region_id: u64,
     pub start_ts: u64,
     pub task_id: u64,
     pub resource_group_tag: Vec<u8>,
+
+    /// Information recorded after the task is scheduled.
```

Comment:
```suggestion
    // Information recorded after the task is scheduled.
```
Or it looks like a doc comment for `request_type` only